geomesa-utils/pom.xml

```diff
@@ -157,6 +157,17 @@
                 </includes>
             </resource>
         </resources>
+
+        <plugins>
+            <plugin>
+                <groupId>pl.project13.maven</groupId>
```

Comment:
Does moving the git-commit-id-plugin to utils mean that all the fat jars end up with a geomesa.properties file?